Readme.md

```diff
@@ -0,0 +1,32 @@
+# trfind
+
+Trip report search tool used by ClimbPlan, hosted at https://climbplan.com.
+
+# Contributing
+
+This is mostly a weekend project, Pull requests are welcome!
+
+## Wishlist
+
+Ideas are tracked using the (issues page)[https://github.com/thatneat/trfind/issues].
+
+## Setup and Running
+
+* Clone the github repo
+* `python3 -m venv env`
+* `source env/bin/activate`
+* `pip install -r requirements.txt`
+
+### CLI Testing
+* `pip install --no-deps .`
+* `trfind 48.51152 -121.05789 "Forbidden Peak"`
```

Comment:
you could also do #11 by documenting how to use `tox` for unit testing

Reply:
added